python/kfserving/docs/KnativeVolatileTime.md

```diff
@@ -3,7 +3,7 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**inner** | [**V1Time**](V1Time.md) |  | 
```

Comment:
Need to double confirm here, is that caused by knative changes? if so, seems the V1Time.md and related code no longer need?

Reply:
Seems that's updated when we generate 0.3 version in https://github.com/kubeflow/kfserving/pull/731, so I think the V1Time.md and v1_time.py is useless, right? of cause you can choose to keep that for safe considering. Thanks.

Reply:
yes, we can delete delete v1_time.py now, the field should be just a datetime type. Knative creates VolatileTime mainly because they want to make it semantically equal when comparing values https://godoc.org/knative.dev/pkg/apis#VolatileTime.